core/templates/base-components/base-content.directive.ts

```diff
@@ -38,9 +39,9 @@ angular.module('oppia').directive('baseContent', [
       template: require('./base-content.directive.html'),
       controllerAs: '$ctrl',
       controller: ['$rootScope', '$window', 'BackgroundMaskService',
-        'SidebarStatusService', 'UrlService',
+        'SidebarStatusService', 'UrlService', 'LoaderService',
```

Comment:
Add the LoaderService in sorted order.

Reply:
Right, this has been fixed... I would also like to point out that the linter couldn't catch this one.

Reply:
@bansalnitish There is something wrong with this check, it is not behaving as expected. I will try to fix this asap.

Reply:
@bansalnitish Currently, the sorted imports check only checks for sorted imports for the top-level directive/controller/factory, we do not have lint check for the lower level. The issue is already filed in #8423

Reply:
Okay sounds good.